src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -28,6 +30,8 @@
  * @author Nicolas Grekas <p@tchwork.com>
  *
  * @final since Symfony 4.3
+ *
+ * @deprecated since Symfony 4.4, use Symfony\Component\ErrorHandler\ExceptionHandler instead.
  */
 class ExceptionHandler
```

Comment:
this class is not restored to it's pre-ErrorRenderer state but it should, isn't it?